Apeer/fiji_scripts/fijipytools.py

```diff
@@ -633,27 +626,26 @@ class ThresholdTools:
 
     @staticmethod
     def apply_autothreshold(hist, method='Otsu'):
-
-        if method == 'Otsu':
-            lowthresh = Auto_Threshold.Otsu(hist)
-        if method == 'Triangle':
-            lowthresh = Auto_Threshold.Triangle(hist)
-        if method == 'IJDefault':
-            lowthresh = Auto_Threshold.IJDefault(hist)
-        if method == 'Huang':
-            lowthresh = Auto_Threshold.Huang(hist)
-        if method == 'MaxEntropy':
-            lowthresh = Auto_Threshold.MaxEntropy(hist)
-        if method == 'Mean':
-            lowthresh = Auto_Threshold.Mean(hist)
-        if method == 'Shanbhag':
-            lowthresh = Auto_Threshold.Shanbhag(hist)
-        if method == 'Yen':
-            lowthresh = Auto_Threshold.Yen(hist)
-        if method == 'Li':
-            lowthresh = Auto_Threshold.Li(hist)
-
-        return lowthresh
+        # I think the use of a dict here makes it a little easier to see what's going on, and avoids repetition
```

Comment:
Again, good catch. I would however remove the comment itself here.

Reply:
I think I've modified everything you've asked me to take out... Apologies for the delay, this is my first pull request, so I'm kind of feeling my way around. Thanks for your patience :)